### PR TITLE
OCMify and cleanup build

### PR DESCRIPTION
**Pull request by ccwienk**

### CODE REVIEW

File: .ci/build_python_packages [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
-repo_dir="$(readlink -f "$(dirname "$0")/..")"
+repo_dir="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")/..")"
 out_dir="$(readlink -f "${repo_dir}/../${PACKAGES_PATH:-}/dist")"
 mkdir "${out_dir}"
 
+ocm_repo='europe-docker.pkg.dev/gardener-project/releases'
+cc_utils_version="$(cat "${repo_dir}/CC_UTILS_VERSION")"
+
+# retrieve packages from cc-utils from OCM (prefer over publishing via PYPI)
+for resource in gardener-cicd-libs gardener-oci; do
```
Comment (zkdev) [inline .ci/build_python_packages:13]: not too happy about hard-coding resource names here.
how about adding meaningful labels, e.g. `python-package`, to corresponding resources and filter by label?

Comment (ccwienk) [reply .ci/build_python_packages:13]: interestingly, I had the same thought (albeit I was more thinking about using a different mimetype for `type` attr).

However, I decided against this. Rationale:

if upstream changes, it is a _good_ thing consumers will break (so changes will not go by unnoticed). Let us assume upstream adds (in other words: we add) a new package. Why would we blindly want to import it (e.g. whd-package, which is already present as of today + which we do not need here)?

/cc @8R0WNI3 wdyt?

Comment (8R0WNI3) [reply .ci/build_python_packages:13]: I'd agree with @ccwienk here, we should better not blindly install all available packages, but only those we explicitly configured.

**Review by 8R0WNI3 — APPROVED** [review]
/lgtm

File: .ci/build_python_packages (hunk 1 above)
Comment (ccwienk) [reply .ci/build_python_packages:13]: btw, we do have labels in place, I think (still this does not change my reasoning) :-)

**Pull request merged**